lib/adapters/test-result/testplane.ts

```diff
@@ -90,18 +92,20 @@ export class TestplaneTestResultAdapter implements ReporterTestResult {
         return new this(testResult, options);
     }
 
-    constructor(testResult: TestplaneTest | TestplaneTestResult, {attempt, status}: TestplaneTestResultAdapterOptions) {
+    constructor(testResult: TestplaneTest | TestplaneTestResult, {attempt, status, duration}: TestplaneTestResultAdapterOptions) {
         this._testResult = testResult;
         this._errorDetails = null;
-        this._timestamp = (this._testResult as TestplaneTestResult).timestamp ??
-            (this._testResult as TestplaneTestResult).startTime ?? Date.now();
+        this._timestamp = (this._testResult as TestplaneTestResult).startTime
+            ?? (this._testResult as TestplaneTestResult).timestamp
```

Comment:
Why do you look at `startTime` first now? Previously you get `timestamp` value first.

Reply:
_timestamp stores startTime, so it makes sense to prioritize startTime over timestamp, i think